base/deprecated.jl

```diff
@@ -2168,6 +2168,11 @@ end
 
 @deprecate merge!(repo::LibGit2.GitRepo, args...; kwargs...) LibGit2.merge!(repo, args...; kwargs...)
 
+# Remember to delete the module when removing this
```

Comment:
You could just move the whole definition here:
```julia
@eval Base.Math module JuliaLibm
    Base.@deprecate log Base.log
end
```

Reply:
Wil fix!